test/Microsoft.AspNet.Mvc.Core.Test/ApplicationModel/DefaultActionModelBuilderTest.cs

```diff
@@ -321,6 +322,27 @@ public void GetActions_ConventionallyRoutedAction_WithHttpConstraints()
         }
 
         [Fact]
+        public void GetActions_AuthorizeFiltersAreCombinedWhenOverriden()
```

Comment:
This isn't really accurate is it? 2 filters are created and both with run so how are they combined?